src/main/java/io/phdata/streamliner/schemadefiner/mapper/SnowflakeMapper.java

```diff
@@ -96,25 +123,37 @@ private static SnowflakeTable mapSchemaCrawlerTable(Table table, Snowflake desti
 
     return new SnowflakeTable(
         table.getName(),
-        getSnowflakeTableName(table.getName(), destination),
+        getSnowflakeTableName(table.getName()),
         table.getRemarks(),
         primaryKey,
         null,
         null,
         null,
-        new FileFormat(table.getName(), "PARQUET"),
+        createFileFormat(table.getName()),
```

Comment:
Is there a reason not to put the file format in the generic properties like in the task? The file format doesn't make sense for other sources

Reply:
Agree. But fileFormat might be needed for the snowflake. By default, we set the location as tableName and fileType as PARQUET. Also, user can override this using [userDefinedTable](https://docs.customer.phdata.io/docs/streamliner/latest/user-manual/#snowflake-user-defined-table) property under source. 
We already have a `fileFormat` field in the state file, I am mapping the location and fileType from hive to the same field.